semgrep/tests/e2e/test_metavariable_regex_const_prop.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+
+@pytest.mark.kinda_slow
```

Comment:
I'm curious about what makes this test "kinda slow".

Reply:
Nothing in particular, in fact it probably isn't slow. But as far as I can tell every other test in the repo is either marked `kinda_slow` or `slow`, so I just went with that.